vis-lua.c

```diff
@@ -1367,6 +1371,55 @@ static int redraw(lua_State *L) {
 	vis_redraw(vis);
 	return 0;
 }
+/***
+ * Closes a stream returned by @{Vis.communicate}.
+ *
+ * @function close
```

Comment:
This function needs to be added to `vis_lua[]`.

Reply:
Actually, no. It is not a function, it is a method. So it is already exposed by setting it to a `closef` member of the ProcessStream structure.

Reply:
Sorry that was my bad. So should it be used like:
```
s = vis:communicate(...)
s:close()
```

Reply:
That is right. The return value for `vis:communicate()` behaves exactly like any other `FILE*` object (e.g. returned by `io.open`)